backend/src/main/java/se/chalmers/dat265/group1/api/digitalpart/DigitalPartController.java

```diff
@@ -3,6 +3,13 @@
 import se.chalmers.dat265.group1.api.ApiController;
 import se.chalmers.dat265.group1.model.DigitalPart;
 import org.apache.commons.lang3.NotImplementedException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.NotImplementedException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)